introduction.md

```diff
@@ -0,0 +1,2 @@
+Hi! My Name Is Cristiano de Oliveira Passos, and I have to make a long sentence....
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.